### PR TITLE
[2.19.x] G-1395 refactor theme scaling to be more intuitive

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
Instead of an arbitrary 62-200% range, this PR adjusts to the range to visibly show 1-100% while preserving the functionality.
#### What are the relevant tickets?
Fixes: #____

#### Screenshots

Before
Screen Shot 2020-08-03 at 11 14 13 AM
Screen Shot 2020-08-03 at 11 14 01 AM
After
Screen Shot 2020-08-03 at 11 13 31 AM
Screen Shot 2020-08-03 at 11 13 22 AM

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
> ❔ Is this worth porting to Revelio or is it handled completely different there?

because of the lack of value, we won't be including this feature in Revelio.   i believe it has already been removed in later versions of Intrigue

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/theme-settings/theme-settings.tsx [hunk 1]
```diff
@@ -76,7 +76,12 @@ const ThemeCustom = styled.div`
 `
 
 const saveFontChanges = (fontSize: string) => {
-  getPreferences().set('fontSize', ThemeUtils.getFontSize(fontSize))
+  getPreferences().set(
+    'fontSize',
+    ThemeUtils.getFontSize(
+      ThemeUtils.getCalculatedZoomScale(fontSize, 1, 100, 62, 200)
```
Comment (rymach) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/theme-settings/theme-settings.tsx:82]: I noticed you `floored` the value from `getCalculatedZoomScale` above, do you need to do that here too?

**Comment by rymach** [comment]
how are you migrating existing zoom level preferences

**Comment by rymach** [comment]
> how are you migrating existing zoom level preferences

never mind

**Comment by rymach** [comment]
isn't this weird? 100% zoom should actually be 100 times larger than 1% right?

**Comment by rymach** [comment]
enhance 
image

**Comment by rymach** [comment]
default zoom should be 100% imo

**Pull request closed without merging**